backend/fms_core/services/library.py

```diff
@@ -145,41 +149,44 @@ def convert_library(process, platform, sample_source, container_destination, coo
 
     return library_destination, errors, warnings
 
+
 def update_library(sample, **kwargs):
```

Comment:
You don't normally want to overwrite a whole bunch of different libraries with the same values. In fact it might be wise to remove the service altogether, or replace the signature by update_library(derived_sample, **kwargs) to be used during capture or inside a controlled loop (applying the change while testing the content of the current library). At the moment the only thing using this is lib_qc to change lib_size so i would test for pool outside and call this service with the appropriate derived_sample instead. We can keep the single library action here instead of the loop. It makes more sense i think.